include/userobjects/BooleanComboClusteringUserObject.h

```diff
@@ -53,9 +53,9 @@ class BooleanComboClusteringUserObject : public GeneralUserObject
   std::vector<std::string> _output_stack;
 
   /// operator predecence
-  static const std::unordered_map<std::string, int> _precedence { {"not", 3}, {"!", 3}, {"and", 2}, {"&&", 2}, {"or", 1}, {"||", 1} };
+  std::unordered_map<std::string, int> _precedence { {"not", 3}, {"!", 3}, {"and", 2}, {"&&", 2}, {"or", 1}, {"||", 1} };
```

Comment:
I think this should still be static, but we may need to define at the top of the implementation (.C) file